src/style.css

```
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+.html {
+  --header-link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+}
+
+.img {
+  display: block;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list__item {
```

```suggestion
.nav__item {
```
You don't need to add one more part of name for each nested element